dxp/demystifying_aui_script_and_liferay-frontend_component_tags.md

```diff
@@ -0,0 +1,114 @@
+# Demystifying `aui:script` and `liferay-frontend:component` tags
+
+On JSP context you may notice a huge number of `aui:script` and `liferay-frontend:component` tags.
+
+## `aui:script` tag
```

Comment:
Should I mention We are moving from using using JS code within our JSPs?

https://github.com/liferay-frontend/liferay-portal/pull/407

Reply:
No need